koku/masu/test/external/downloader/oci_local/test_oci_local_report_downloader.py

```diff
@@ -82,7 +82,9 @@ def test_initializer(self):
         """Test the OCI-Local initializer."""
         self.assertIsNotNone(self.report_downloader)
 
-    def test_download_file(self):
+    @patch("masu.util.aws.common.remove_files_not_in_set_from_s3_bucket")
+    @patch("masu.util.aws.common.copy_data_to_s3_bucket")
+    def test_download_file(self, _, __):
```

Comment:
I think you get the idea.
```suggestion
    def test_download_file(self, *args):
```